### ISSUE TITLE
Change uwgsi.xml uid to 1014

### ISSUE THREAD

**Issue by nukeador** [mozillahispano/mozbuzz#46]
We are isolating each web with a different user and we need mozbuzz to be under uid 1014.

This should be applied on production branch.

**Comment by nukeador** [comment]
Don't apply this yet. We are figuring out some stuff on server side to make it works.

**Comment by sebasmagri** [comment]
@nukeador any news on this matter?

**Comment by nukeador** [comment]
No, @willyaranda do we know how to do this safely per virtualhost?

**Comment by willyaranda** [comment]
Not really, but the idea would be:
- Add a `<uid>www-data</uid>` and `<gid>www-data</gid>` to the `uwsgi.xml`, and make sure all directory is readable by that user.
- Make that nginx is capable of reading the socket (it can be put on `/tmp` for example)

But I don't have any free time this week.

**Comment by sebasmagri** [comment]
@nukeador @willyaranda is this still necessary now that we're deploying a wsgi script and recommended uwsgi setup?

**Comment by nukeador** [comment]
Yes, we want uwgsi to run under a specific user but we need time to figure out how.

Can we put this on-hold?
